routers/repo/attachment.go

```diff
@@ -124,21 +124,26 @@ func GetAttachment(ctx *context.Context) {
 		}
 	}
 
+	if err := attach.IncreaseDownloadCount(); err != nil {
```

Comment:
Is it ok to move `IncreaseDownloadCount` up? Now the download is counted even if an error may occur. In the old code the redirect or `ServeData` may fail too.

Reply:
I'm fine with that.